integration-cli/docker_cli_version_test.go

```diff
@@ -26,3 +26,28 @@ func (s *DockerSuite) TestVersionEnsureSucceeds(c *check.C) {
 		}
 	}
 }
+
+// ensure the Windows daemon return the correct platform string
+func (s *DockerSuite) TestVersionPlatform_w(c *check.C) {
+	testRequires(c, DaemonIsWindows)
+	testVersionPlatform(c, "windows/amd64")
+}
+
+// ensure the Linux daemon return the correct platform string
+func (s *DockerSuite) TestVersionPlatform_l(c *check.C) {
+	testRequires(c, DaemonIsLinux)
+	testVersionPlatform(c, "linux/amd64")
+}
+
+func testVersionPlatform(c *check.C, platform string) {
+	out, _ := dockerCmd(c, "version")
+
+	// Verify the second 'OS/Arch' matches the platform
+	split := strings.Split(out, "\n")
+	if len(split) != 16 {
```

Comment:
@calavera - the line check is there to ensure the index of split[14] dpesn't AV. As we know the answer is on line 14, this IMO was the most expedient way of getting that info. I can't do a simple strings.Contains on out as there are two platform/arch strings in the output of docker version....